### PR TITLE
Fixes VSTS Bug 790861: Text not redrawn when auto-formatting

### PR DESCRIPTION
**Pull request by mkrueger**
https://devdiv.visualstudio.com/DevDiv/_workitems/edit/790861

### CODE REVIEW

**Comment by mkrueger** [comment]
@monojenkins backport master

**Comment by mkrueger** [comment]
Fixes the issue as well is a perf. improvement. There is basically no reason to check it that way, setting a flag there is easier.

File: main/src/addins/MonoDevelop.SourceEditor2/VSEditor/TagBasedSyntaxHighlighting.cs [hunk 1]
```diff
@@ -203,20 +203,11 @@ private void OnClassificationChanged (object sender, ClassificationChangedEventA
...
-						handler (this, new LineEventArgs (line.line));
-						continue;
-					}
-					for (int i = 0; i < oldSegments.Count; i++) {
-						if (newSegments [i].ColorStyleKey != oldSegments [i].ColorStyleKey) {
-							handler (this, new LineEventArgs (line.line));
-							break;
-						}
-					}
+					if (!line.HasDrawn) {
```
Comment (Therzok) [inline main/src/addins/MonoDevelop.SourceEditor2/VSEditor/TagBasedSyntaxHighlighting.cs]: What resets this though?

Comment (mkrueger) [reply main/src/addins/MonoDevelop.SourceEditor2/VSEditor/TagBasedSyntaxHighlighting.cs]: Basically that fires a redraw when it's newly created - default value is false.

**Pull request merged**